Zend/zend_ast.c

```diff
@@ -472,6 +473,9 @@ static void zend_ast_destroy_ex(zend_ast *ast, zend_bool free) {
 			zend_ast_destroy_ex(decl->child[1], free);
 			zend_ast_destroy_ex(decl->child[2], free);
 			zend_ast_destroy_ex(decl->child[3], free);
+			if (decl->child[4]) {
+				zend_ast_destroy_ex(decl->child[4], free);
+			}
```

Comment:
Maybe, have a look at assert code and have a play with it ... I'm not sure it was really meant for that, and it may not always be "pretty", but it will be consistent .. but making it stick to some standard may be quite difficult .. I'm not sure ...